packages/service/src/modules/slack/sync-users-service.ts

```diff
@@ -10,25 +10,51 @@ export class SyncSlackUsersService {
     this.web = new WebClient(getEnvVariable('SLACK_TOKEN'));
   }
 
-  public async syncUsers() {
+  private async slackUsers() {
+    return (await this.web.users.list({ team_id: getEnvVariable('SLACK_TEAM_ID') })).members as Member[];
+  }
+
+  /**
+   * Updates a user's slack handle, if the user is found. Otherwise does nothing.
+   * @param {String} name name of user
+   * @param {String} email email of user
+   * @param {String} slackHandle user's slack handle
+   */
+  private async updateSlackHandle(name: string, email: string, slackHandle: string) {
     const { repositories, logger } = this.context;
 
-    const slackUsers = (await this.web.users.list({ team_id: getEnvVariable('SLACK_TEAM_ID') })).members as Member[];
+    const foundUser = await repositories.user.findByEmail(email);
+    if (!foundUser) {
+      return;
```

Comment:
Här bör du i stället throwa ett error, t ex `throw new BadRequestError('User not found')`